.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "pytorch_xla",
+  "image": "us-central1-docker.pkg.dev/tpu-pytorch/docker/development:tpu",
+  "runArgs": [
+    "--privileged"
+  ],
+  "forwardPorts": [
+    4200
+  ],
+  "initializeCommand": "docker pull us-central1-docker.pkg.dev/tpu-pytorch/docker/development:tpu",
```

Comment:
Does this update your local copy of the container to the latest one?

Reply:
Yes, this command runs before everything else.